data/autoyast_hanaperf/sle15.xml.ep

```diff
@@ -109,7 +113,7 @@
       <resolv_conf_policy>auto</resolv_conf_policy>
       <write_hostname config:type="boolean">false</write_hostname>
     </dns>
-    <ipv6 config:type="boolean">false</ipv6>
```

Comment:
why we need this change?

Reply:
Originally the autoyast file is used for x86_64 system only, they are located at Beijing lab without IPv6 supported. Recently, we got ppc64le system back, they are located at Prague lab with IPv6 support, so I enable IPv6 support as default option.